web/package.json

```diff
@@ -91,24 +91,24 @@
     "@types/react-redux": "^7.0.6",
     "@types/react-tap-event-plugin": "^0.0.30",
     "@types/redux-logger": "^3.0.7",
-    "@typescript-eslint/eslint-plugin": "^5.13.0",
-    "@typescript-eslint/parser": "^5.13.0",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
+    "@typescript-eslint/parser": "^5.49.0",
     "awesome-typescript-loader": "^5.2.1",
     "clean-webpack-plugin": "^2.0.1",
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.8.0",
     "enzyme-to-json": "^3.3.5",
     "es6-promise": "^4.2.8",
     "eslint": "^8.10.0",
-    "eslint-config-prettier": "^6.3.0",
-    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-prettier": "^3.4.1",
     "eslint-plugin-react": "^7.27.0",
     "eslint-plugin-sort-imports-es6-autofix": "^0.6.0",
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "27.3.1",
-    "prettier": "1.18.2",
+    "prettier": "^1.19.1",
```

Comment:
We're bumping versions here for things not related to this workflow. I'm happy to have a PR that only updates these versions, but I think this could potentially cause unrelated issues that are hard to debug.

Reply:
Updated version for this package to pass tests for new ecmascript features as "optional chaining", because current doesn't support it and throw errors. Do you mean that I can create separate PR for it and it's be okay? Or if I understood wrong, what do you think will be better solution to resolve it? Because it's make code more clean and readable. Thanks

Reply:
Let's make this a seperate PR.

Reply:
Sure, did it, please review https://github.com/MarquezProject/marquez/pull/2393

Reply:
Also reverted changes with versions for this PR.